pmd-javascript/src/main/java/net/sourceforge/pmd/lang/typescript/ast/TypeScriptLexerBase.java

```diff
@@ -7,7 +7,9 @@
 
 package net.sourceforge.pmd.lang.typescript.ast;
 
-import org.antlr.v4.runtime.*;
```

Comment:
this wildcard it nice to see gone, is the unused import now resolving these, or what recipe did it?

@timtebeek